### PR TITLE
added fouc-prevention with body[unresolved]

### PR DESCRIPTION
**Pull request by dotch**

### CODE REVIEW

File: media/css/site.css [hunk 1]
```diff
@@ -3,12 +3,6 @@
   -moz-boz-sizing: border-box;
 }
 
-[unresolved] {
```
Comment (fwenzel) [inline media/css/site.css:6]: what's the point of adding the attribute, then removing the styles for it?

File: usage.html [hunk 2]
```diff
@@ -83,6 +83,9 @@ <h1 id="usage">Usage</h1>
         prototype: RouteLinkProto,
         extends: 'a'
       });
+      document.addEventListener('WebComponentsReady', function(){
+        document.body.removeAttribute("unresolved");
```
Comment (fwenzel) [inline usage.html]: mixing double and single quotes?

**Comment by fwenzel** [comment]
This'll change the FOUC to a FONC (... of no content), but I think sadly there's no other way

File: media/css/site.css (hunk 1 above)
Comment (dotch) [inline media/css/site.css:6]: body[unresolved] {
  opacity: 0;
}

will be injected by platform.js

File: usage.html (hunk 2 above)
Comment (dotch) [inline usage.html]: fixed!

File: media/css/site.css (hunk 1 above)
Comment (fwenzel) [inline media/css/site.css:6]: thanks!

**Comment by potch** [comment]
Maybe we can do a FOSS (flash of splash screen)

**Pull request merged**